go/libkb/saltpack.go

```diff
@@ -138,8 +138,6 @@ func BoxPublicKeyToKeybaseKID(k saltpack.BoxPublicKey) (ret keybase1.KID) {
 }
 
 func checkSaltpackBrand(b string) error {
-	if b != KeybaseSaltpackBrand {
-		return KeybaseSaltpackError{}
-	}
```

Comment:
I remember that Patrick and I wanted to delete this, but Max wanted to keep it. If we keep this check, then the example blob in the spec below will need to include the KEYBASE brand, or else we won't be able to verify it. I don't feel super strongly either way.